src/patterns/components/offline-notification/example.hbs

```diff
@@ -0,0 +1,12 @@
+---
+name: Example
+sourceless: true
+hidden: true
+---
+
+<div class="Message Message--offline">
+  You seem to be offline at the moment. <span class="u-hidden u-md-inlineBlock">We'll keep trying to connect.</span> <a href=".">Try again</a>
+  <a href="#" class="Message-close">
```

Comment:
This should be a `<button>` element since it does not actually resolve to a destination or trigger a reload of the current destination.